audit/src/domain/audit.ts

```diff
@@ -0,0 +1,11 @@
+// Define the interface for the audit repo
+
+export interface AuditLogItem {
+  subject: string;
```

Comment:
@petereast ... don't we need an 'objectId'

Reply:
Isn't that the same as `entity`?